src/utils/getRoutes.js

```diff
@@ -94,6 +100,9 @@ export async function getRoutes() {
     ...createListRoutes('src/templates/Lists', lists, allPages),
     ...createListRoutes('src/templates/Lists', authors, allPages, authorProps),
     ...createRoutes('src/templates/Subpage', blogPosts, allPages, blogPostProps),
+    ...createRoutes('src/templates/Subpage', careers, allPages),
```

Comment:
Need to render the Careers template

```suggestion
    ...createRoutes('src/templates/Careers', careers, allPages),
```